src/main/java/pl/zimi/zadanka/Zad5Drafts.java

```diff
@@ -0,0 +1,76 @@
+package pl.zimi.zadanka;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class Zad5Drafts {
+    public static int climbStairsTreeSet(int n) {
```

Comment:
czy ten kod chcesz zachować w jakimś celu? jeśli tak to spoko ale w taki sposób żeby jasne było po co chcesz go zachować (np. do nauki)

jeśli chcesz go zachować z sentymentu... to go usuń... zadaniem gita jest trzymać stare rzeczy... aktualna wersja powinna trzymać  tylko potrzebne...

Reply:
z sentymentu, dużo się namęczyłem przy nim. Po przejrzeniu nie widzę w nim wartości edukacyjnych, więc usuwam.